docs/DOCKER.md

```diff
@@ -47,7 +47,7 @@ Second, copy the token that you just generate and run docker login command like
 $ docker login docker.pkg.github.com -u USERNAME 
 ```
 
-**6.** Go to the project's root directory and run:
+**6.** Enter to anyway-newsflash-infographics directory and run: `bash run.bash` this will download the anyway-newsflash-infographics repo. then go to the project's root directory and run:
```

Comment:
@Shai2022 looks great. Can you add this to row 99 as well? same instruction.
Also, perhaps we can separate it to 2 different stages (6 and 7, and shift the rest)

Reply:
Done